nl_server/tests/lib/nl_embeddings_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Embeddings (in nl_embeddings.py)."""
+
+from diskcache import Cache
+from lib.nl_embeddings import Embeddings
+from parameterized import parameterized
+
+import unittest
+
+
+class TestEmbeddings(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls) -> None:
+
+    # Look for the Embeddings model in the cache if it exists.
+    nl_embeddings_cache_key = 'nl_embeddings'
+    nl_cache_path = '~/.datacommons/'
```

Comment:
nit: can we export these as constants? here and in the next test

Reply:
Thanks for the suggestion! Made an update: reading as imports now.